src/it/container/start-waves.sh

```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
 
-WAVES_IP=`ifconfig eth0 | awk '/inet addr/ {gsub("addr:", "", $2); print $2}'`
+WAVES_IP=`ifconfig eth1 | awk '/inet addr/ {gsub("addr:", "", $2); print $2}'`
```

Comment:
Because from now the connection between the nodes I use another interface - the network interface, which I created myself and disconnect it during the test. And node api communicates by ip of eth0.